backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/utils/CassandraUtils.java

```diff
@@ -40,6 +41,10 @@ public CassandraUtils(CassandraConfiguration cassandraConfiguration) {
         this.cassandraConfiguration = cassandraConfiguration;
     }
 
+    public Flux<Row> convertToFlux(ResultSet resultSet) {
+        return Flux.fromStream(convertToStream(resultSet));
```

Comment:
Can't we rather leverage the fact that ResultSet is an Iterable? 

This seems better than an intermediate Stream representation...